components/schema-migrator/migrations/director/20220914135000_ord_increase_short_description_size.down.sql

```diff
@@ -0,0 +1,46 @@
+BEGIN;
```

Comment:
Make sure to update the file name as the current latest migration has a timestamp of `20220920145100`.
When we introduce UP and DOWN migrations, they should have the latest timestamp.

Reply:
Renamed the file names with the latest timestamp.